packages/documentation/package.json

```diff
@@ -9,10 +9,10 @@
     "astro": "astro"
   },
   "dependencies": {
-    "@astrojs/starlight": "^0.13.1",
-    "@interledger/docs-design-system": "^0.3.2",
-    "astro": "3.6.5",
-    "astro-graphql-plugin": "^0.1.0",
+    "@astrojs/starlight": "^0.21.5",
+    "@interledger/docs-design-system": "^0.4.0",
+    "astro": "4.6.4",
+    "astro-graphql-plugin": "^0.2.1",
```

Comment:
This update wasn't followed up with an update to the lockfile which is why I think the build is failing.